src/shared/ui/widget-title/widget-title.tsx

```diff
@@ -9,33 +9,33 @@ type WidgetTitleProps = HTMLAttributes<HTMLHeadingElement>
 
 export const cx = classNames.bind(styles);
 
-const widgetTitleVariants = cva(cx('title'), {
+const widgetTitleVariants = cva(cx('widget-title'), {
   variants: {
-    mods: {
-      lines: cx('lines'),
-      asterisk: cx('asterisk'),
-    },
     size: {
       small: cx('small'),
       medium: cx('medium'),
       large: cx('large'),
     },
+    mods: {
+      lines: cx('lines'),
+      asterisk: cx('asterisk'),
+    },
   },
   defaultVariants: {
     size: 'medium',
     mods: null,
   },
 });
 
-export const WidgetTitle = ({ children, size, mods, className, ...props }: WidgetTitleProps) => {
```

Comment:
Same as https://github.com/rolling-scopes/site/pull/419#discussion_r1699504907

Reply:
I mean `type WidgetTitleProps = VariantProps<typeof widgetTitleVariants>;`

Reply:
We've an error here 
![image](https://github.com/user-attachments/assets/a3f51b46-a413-46e2-baa7-c92781426e47)

Reply:
I see two ways to fix this
```
type WidgetTitleProps = Pick<HTMLAttributes<HTMLHeadingElement>, 'className' | 'children'> &
  VariantProps<typeof widgetTitleVariants>;
```

```
type WidgetTitleProps = { children: React.ReactNode;
  className?: string; } & VariantProps<
  typeof widgetTitleVariants
  >;
```